pkg/config/config.go

```diff
@@ -194,3 +207,72 @@ func LoadV2(path string) *ConfigMap {
 	cfg := parseV2(configPath)
 	return cfg
 }
+
+type Client struct {
+	client *etcdv3.Client
+}
+
+func NewClient(endpiont []string) (*Client, error) {
+	tmpClient, err := etcdv3.NewConfigClientWithErr(
+		etcdv3.WithName(etcdv3.RegistryETCDV3Client),
+		etcdv3.WithTimeout(10*time.Second),
+		etcdv3.WithEndpoints(endpiont...),
+	)
+	if err != nil {
+		return nil, fmt.Errorf("Init etcd client fail error %v", err)
```

Comment:
How about use github.com/pkg/errors instead? For example: pkgErrors.Wrapf(err, "....")